tests/nvector_destroy.cc

```diff
@@ -0,0 +1,17 @@
+#include "nvector_eigen.h"
+#include "eigen3/Eigen/Dense"
+#include <iostream>
+
+using Vector = Eigen::Matrix<realtype, Eigen::Dynamic, 1>;
+
+int main ()
+{
+  N_Vector v = create_eigen_nvector<Vector>(2);
+
+
+  v->ops->nvdestroy(v);
+  // FIXME For some reason v->ops and v->content are not set to nullptr. I don't understand why.
+  std::cout << (v->ops == nullptr) << std::endl;
+  std::cout << (v->content == nullptr) << std::endl;
```

Comment:
You are outputting a `bool` here, but C++ says that that is output as 0=false and 1=true. So if you see a `1` in the output, this is what you actually want I guess.

You might want to look at https://en.cppreference.com/w/cpp/io/manip/boolalpha

Reply:
Yeah I was basing it off `1==true` and `0==false` but using `std::boolalpha` makes it easier!